packages/jss-preset-default/src/index.js

```diff
@@ -0,0 +1,40 @@
+// @flow
```

Comment:
if you add flow as a separate step, you will speed up merging

Reply:
I added flow here already because it's a really small change. I bigger plugins I will definitely create a separate PR for flow